site/class-1-bar/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
You’re uploading files to `site/class-1-bar/` instead of `site/class-1-bar/larsmarjew`!

Reply:
...and you’re adding both your bar *and* play assignment in at!
Maybe it’s simplest to start over, creating two new branches and PRs?